pravega_provisioner/cluster_provisioner.py

```diff
@@ -257,8 +302,19 @@ def main():
 
     if provisioning_model == 0:
         vms = int(input("How many VMs/nodes do you want to devote for a Pravega cluster?"))
-        zookeeper_servers, bookkeeper_servers, segment_stores, controllers = resource_based_provisioning(vms, vm_cpus,
-                        vm_ram_gb, vm_local_drives, zookeeper_servers, bookkeeper_servers, segment_stores, controllers)
+        # Ask to the user the number of node failures he/she wants to tolerate.
+        failures_to_tolerate = get_int_input("How many node failures you want to tolerate?", range(0, 2))
```

Comment:
Please, leave the range of failures to what it was, as people may want to deploy Pravega on huge environments in which experiencing more than 2 failures is a common scenario.

Reply:
Can we change to 10 instead of original value 100. I think Ashish also asked to reduce this.
please see comment 1 in https://asdwiki.isus.emc.com:8443/display/NAUT/Pravega+Cluster+Provisioning+Model

Reply:
I saw the comment, and I don't see the point of limiting the script to 2 or 10 failures. Is just a modeling script, no something related to any infrastructure.